requirements/run.txt

```diff
@@ -95,7 +103,7 @@ traitlets==5.3.0
     # via
     #   ipython
     #   matplotlib-inline
-typing-extensions>=4.0.1
+typing-extensions==4.4.0
```

Comment:
`typing-extensions` ensures backwards compatibility with its major version, so let's keep as we used to have here to facilitate.

Reply:
Changed.